src/server.py

```diff
@@ -24,6 +37,12 @@ def api_top_image():
         "title": article.title,
         "topimage": article.top_image}), 200, {'Content-Type': 'application/json'}
 
+
+def is_content_pdf(url):
+    response = requests.head(url)
+    return response.headers['Content-Type'] == 'application/pdf'
```

Comment:
It's possible that server won't return this header, for example some static file server might return the file without this header.

Reply:
this logic has been removed from our codebase